app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    test = split_integer(16, 4)
+    assert sum(test) == 16
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    test = split_integer(6,2)
+    assert test == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    test = split_integer(8, 1)
+    assert sum(test) == 8
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    test = split_integer(17, 4)
+    assert test == [4, 4, 4, 5]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    test = split_integer(4, 5)
```

Comment:
As per the checklist, avoid using unnecessary variables. Directly assert the result of `split_integer(4, 5)`.